lib/rules/padding-line-between-tags.js

```diff
@@ -109,40 +154,38 @@ function checkNewline(context) {
         const lineDifference =
           closestSibling.loc.start.line - endTag.loc.end.line
         if (configure.blankLine === 'always') {
-          if (lineDifference === 1) {
-            insertNewLine(context, block, closestSibling)
-          } else if (lineDifference === 0) {
-            context.report({
-              messageId: 'always',
-              loc: closestSibling.loc,
-              // @ts-ignore
-              fix(fixer) {
-                const lastSpaces = /** @type {RegExpExecArray} */ (
-                  /^\s*/.exec(
-                    context.getSourceCode().lines[endTag.loc.start.line - 1]
-                  )
-                )[0]
-
-                return fixer.insertTextAfter(endTag, `\n\n${lastSpaces}`)
-              }
-            })
-          }
-        } else {
-          if (lineDifference > 1) {
-            let hasOnlyTextBetween = true
-            for (
-              let i = endTag.loc.start.line;
-              i < closestSibling.loc.start.line - 1 && hasOnlyTextBetween;
-              i++
-            ) {
-              hasOnlyTextBetween = !/^\s*$/.test(
-                context.getSourceCode().lines[i]
-              )
+          insertNewLine(context, block, closestSibling, lineDifference)
+        } else if (configure.blankLine === 'consistent') {
+          let newlineCount = 0
+          let siblingLineDifference = 0
+          /** @type {VElement | null} */
+          let prevChild = null
+          const siblingElements = block.parent.children.filter(
+            (element) => element.type === 'VElement' && hasConsistentConfiguration(element, configureList)
+          )
+          // get parent
+          for (const child of siblingElements) {
+            if (!prevChild) {
+              prevChild = /** @type {VElement} */ (child)
+              continue
             }
-            if (!hasOnlyTextBetween) {
-              removeExcessLines(context, endTag, closestSibling)
+            siblingLineDifference =
+              child.loc.start.line - prevChild.loc.end.line
+
+            if (siblingLineDifference > 1) {
+              newlineCount++
             }
+            prevChild = /** @type {VElement} */ (child)
           }
+          const ratio = newlineCount / (siblingElements.length - 1)
+
+          if (0.5 - ratio < 0.005) {
```

Comment:
I think it would be simpler to check if it matches the first line break style instead of the ratio. Users can switch styles using auto-fix after changing the first one to their preferred line break style.